src/flatbuffer_types.rs

```diff
@@ -78,6 +78,7 @@ impl FlatBufferSsTableInfoCodec {
             filter_offset: info.filter_offset(),
             filter_len: info.filter_len(),
             compression_codec: info.compression_format().into(),
+            row_attributes: vec![RowAttribute::Flags],
```

Comment:
eventually we will read this from the encoded file, but for now we just hardcode the row_attributes to be just Flags